src/Main.cpp

```diff
@@ -81,9 +81,10 @@ int usage()
         << "  reference-station PORT TARGET        configure itself as a RTK reference\n"
         << "                                       station, forward RTCM messages to TARGET\n"
            "TARGET\n"
-        << "  chrony-configure PORT\n"
+        << "  chrony-configure PORT [PERIOD_US]\n"
         << "     permanently configure the device to output the UBLOX time message\n"
-        << "     on PORT that is needed for the chrony subcommand\n"
+        << "     on PORT that is needed for the chrony subcommand. Optionally modify\n"
+        << "     the period of the PPS (in microseconds)\n"
```

Comment:
You could include that the standard period is 1second

Reply:
The intention was to not modify the period at all. Fixed both documentation and code.